test/test_torch.py

```diff
@@ -14692,6 +14692,35 @@ def test_ctor_with_numpy_array(self, device):
             for i in range(len(array)):
                 self.assertEqual(tensor[i], array[i])
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    @dtypes(*torch.testing.get_all_dtypes())
+    def test_numpy_scalar_cmp(self, device, dtype):
+        if dtype.is_complex:
```

Comment:
@mruberry do we have any function similar to `_make_tensors` in `TestTorchDeviceType`?

Reply:
Recap of offline discussion: https://github.com/pytorch/pytorch/pull/43451 is adding a `make_tensor` method to common_utils.py.